File: src/main/java/net/zoe/corporation/commands/utils/PingCommand.java
package net.zoe.corporation.commands.utils;

import net.zoe.corporation.structures.commands.Command;
import net.zoe.corporation.structures.commands.CommandContext;

import java.util.List;

public class PingCommand extends Command {
    public String name = "ping";
    public List<String> aliases = List.of("latency");
    public String description = "Show my latency";
    public String usage = "<prefix>ping [clusters/shards]";
    public String category = "Utils";
    public boolean canDisable = false;
    public int cooldown = 5;

    @Override
    public void execute(CommandContext ctx) {
        ctx.getJDA().getRestPing();
    }
}

File: src/main/kotlin/com/github/mrdroox/projects/zoe/commands/utils/PingCommand.kt
package com.github.mrdroox.projects.zoe.commands.utils

import com.github.mrdroox.projects.zoe.structures.Command
import com.github.mrdroox.projects.zoe.structures.CommandContext

class PingCommand: Command() {
    override var name: String? = "ping"
    override var aliases: List<String> = listOf("latency", "apiping")
    override var description: String = "commands:$name.description"
    override var usage: String = "commands:$name.usage"
    override var category: String = "commands:$name.category"
    override var canDisable: Boolean = false

    override fun execute(ctx: CommandContext) {
        ctx.textChannel?.sendMessage("aaa")?.queue()
    }
}